draft-ietf-rats-architecture.md

```diff
@@ -130,43 +124,44 @@ Appraisal Policy for Attestation Result:
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the Evidence it has appraised
 
 Attester:
 
-: An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
+: An entity (typically a device), whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation
 
 Claim:
 : A piece of asserted information, often in the form of a name/value pair.
 (Compare /claim/ in {{RFC7519}})
 
 Endorsement:
 
-: A secure statement that some entity (typically a manufacturer) vouches for the integrity of an Attester's signing capability
+: Statements that Endorsers make (typically a manufacturer) that vouches for the design and implementation of the Attester. Often this includes statements about the integrity of an Attester's signing capability
```

Comment:
I think this change is less correct than the original text.  The point about the "signing capability" typically refers to the Attesting Environment, and the endorsement vouches for that environment but not the target environment.   The new proposed text implies that it also vouches for target environments, which I think is not correct.  Recommend reverting.

Reply:
After reading this more thoroughly again, I have to agree with Dave here, too.